src/Controller/MatchController.php

```diff
@@ -19,19 +20,27 @@ public function index(
         Request $request
     ) {
         $this->denyAccessUnlessGranted('IS_AUTHENTICATED_FULLY');
-        $matchesPagination = [];
-        if ($this->getUser()->getStatus() === 'active') {
-            $service->filter($this->getUser());
-            $matches = $service->getPossibleMatch($this->getUser());
-            $matchesPagination = $ps->getPagerfanta($matches);
-            $matchesPagination->setMaxPerPage(8);
-            $matchesPagination->setCurrentPage($request->query->get('page', 1));
+
+        $form = $this->createForm(MatchFilterFormType::class);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            var_dump($form->getData());
+            if ($this->getUser()->getStatus() === 'active') {
```

Comment:
'active' turetu but constanta `User` entityje. o is vis geriausia turet metoda $user->isActive(). Visada venk tokiu magic stringu ir skaiciu